.github/workflows/ci-unit-tests.yml

```diff
@@ -14,6 +14,8 @@ concurrency:
 # See https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions
 permissions:  # added using https://github.com/step-security/secure-workflows
   contents: read
+  checks: write
+  pull-requests: write
```

Comment:
Do we need both of these? The action is not supposed to change the PR itself, so I am suspicious about the 2nd entry

Reply:
Just following instructions from [here](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v2.12.0/README.md#permissions).

Actually, I think you're right; I reckon they relate to adding the PR comment and check summary. Removed in [2be51fa](https://github.com/jaegertracing/jaeger/pull/5030/commits/2be51faa33556599bc44746e176102c958e705a4).